forester/tests/test_indexer.rs

```diff
@@ -50,20 +57,18 @@ async fn test_indexer() {
         env.transfer_sol(user_index).await;
     }
 
-    let indexer = env.indexer;
-
-    assert_ne!(get_state_queue_length(&config), 0);
+    assert_ne!(get_state_queue_length(&mut env.rpc, &config).await, 0);
     info!(
         "Nullifying queue of {} accounts...",
-        get_state_queue_length(&config)
+        get_state_queue_length(&mut env.rpc, &config).await
     );
-    nullify(indexer, &config).await.unwrap();
-    assert_eq!(get_state_queue_length(&config), 0);
+    let _ = nullify(&mut env.indexer, &mut env.rpc, &config).await;
```

Comment:
why not use the result here?
We are awaiting the transaction so I don't see why we shouldn't fail if the transaction fails.

Reply:
Fixed.